R/class2tree.R

```diff
@@ -180,56 +184,60 @@ taxa2dist <- function(x, varstep = FALSE, check = TRUE, labels) {
   out
 }
 
-###############################################################################
-#################### GET LIST OF ALL TAXONOMY RANK AND IDs ####################
-###############################################################################
-get_rank <- function(x){
-  rankDf <- x[, 'rank']
-  names(rankDf) <- x[, 'rank']
-
-  idDf <- x[, 'id']
-  joinedDf <- cbind(data.frame(rankDf,stringsAsFactors=FALSE),
-                    data.frame(idDf,stringsAsFactors=FALSE))
-  joinedDf <- within(joinedDf,
-                     rankDf[rankDf=='no rank'] <-
-                     paste0("norank_",idDf[rankDf=='no rank']))
-
-  df <- data.frame(t(data.frame(rev(joinedDf$rankDf))),
-                   stringsAsFactors = FALSE)
+#' Get full taxonomy ranks and IDs
+#' @author Vinh Tran {tran@bio.uni-frankfurt.de}
```

Comment:
add `#' @noRd` to each of the 4 fxns you added docs to. these fxns aren't meant to be exported to the user, and as far as I can tell the docs would only be for package developers.

Reply:
already added @noRd to the functions

Reply:
thanks